File: Chitunion/Web(java)/src/main/java/com/xyauto/domain/RoleInfo.java
package com.xyauto.domain;

import java.util.Date;

import lombok.Data;

@Data
public class RoleInfo {

	private String roleID;
	private Integer recID;
	private String sysID;
	private String roleName;
	private String intro;
	private Integer status;
	private Date createTime;
	private Integer createUserID;
}

File: Chitunion/Web(code refactoring)/XYAuto.ChiTu2018/XYAuto.ChiTu2018.Entities/Chitunion2017/RoleInfo.cs
namespace XYAuto.ChiTu2018.Entities.Chitunion2017
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("RoleInfo")]
    public partial class RoleInfo
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int RecID { get; set; }

        [Required]
        [StringLength(50)]
        public string SysID { get; set; }

        [Key]
        [StringLength(50)]
        public string RoleID { get; set; }

        [StringLength(100)]
        public string RoleName { get; set; }

        [StringLength(500)]
        public string Intro { get; set; }

        public int? Status { get; set; }

        public DateTime? CreateTime { get; set; }

        public int? CreateUserID { get; set; }
    }
}
